Frontend/ui-library/src/Application/Application.ts

```diff
@@ -46,6 +46,7 @@ import { FullScreenIconBase, FullScreenIconExternal } from '../UI/FullscreenIcon
  */
 export type VideoQPIndicatorConfig = {
     disableIndicator?: boolean;
+    hideWhenGood?: boolean;
```

Comment:
Maybe a comment above this, saying something like:

When this flag is enabled we will hide the QP indicator when the connection is "good" (e.g. when the icon would normally be green).